script/sdbSpecList.sh

```diff
@@ -6,9 +6,9 @@
 # On `sdb devices`, it will show result below.
 #
 # List of devices attached 
-# 192.168.223.1:26101 	device    	0
-# emulator-1            emulator    0
-# 192.168.224.1:26101 	device    	0
+# 1.2.3.4:26101 	device    	0
```

Comment:
I think previous IP address looks better for this example because `1.2.3.4` does  not look like IP addess. You also mentioned that this is serial number.. I am afraid that some readers might think that this is some kind of serial number format of sdb, not IP address.

Reply:
Or you may want to mention that `the first column is some unique string such as serial or IP address`.